react/features/settings/components/web/ProfileTab.js

```diff
@@ -75,8 +75,8 @@ class ProfileTab extends AbstractDialogTab<Props> {
     render() {
         const {
             authEnabled,
-            displayName,
-            email,
+            displayName = '',
```

Comment:
why is this needed? Maybe use `defaultProps` if need be.

Reply:
This was needed to fix a bug on the avatar initials update on editing the displayName (remove it and you'll see the warning generated by Redux). I'll add them to defaultProps though